src/WebUI/locales/en.yml

```diff
@@ -1019,12 +1019,19 @@ item:
     Cut:
       long: 'Cut'
       short: 'c'
+      title: 'Cut damage'
+      description: 'Cut damage description'
     Pierce:
       long: 'Pierce'
       short: 'p'
+      title: 'Pierce damage'
+      description: 'Pierce damage description'
     Blunt:
       long: 'Blunt'
       short: 'b'
+      title: 'Blunt damage'
+      description: 'Blunt damage description'
```

Comment:
Blunt damage refers to the trauma caused by impact weapons like hammers, maces, or clubs. This type of damage is akin to pierce damage in terms of its effectiveness against armoured foes. However, unlike pierce damage, blunt damage does not have an increased headshot multiplier. This makes blunt better for general use comparatively to that of pierce.